src/main/java/com/google/devtools/build/lib/runtime/commands/CleanCommand.java

```diff
@@ -49,14 +53,15 @@
 
 /** Implements 'blaze clean'. */
 @Command(
-  name = "clean",
-  builds = true, // Does not, but people expect build options to be there
-  writeCommandLog = false, // Do not create a command.log, otherwise we couldn't delete it.
-  options = {CleanCommand.Options.class},
-  help = "resource:clean.txt",
-  shortDescription = "Removes output files and optionally stops the server.",
-  // TODO(bazel-team): Remove this - we inherit a huge number of unused options.
-  inherits = {BuildCommand.class}
+    name = "clean",
+    builds = true, // Does not, but people expect build options to be there
+    allowResidue = true, // Does not, but need to allow so we can ignore Starlark options.
```

Comment:
not sure what git diff thinks is happening, I only added line 58

Reply:
The indentation changed (because the formatter considers this one statement to be reformatted when any part of it changes).